### DISCUSSION TITLE
Add support for jumping to quoted message

### DISCUSSION THREAD

**Discussion by Kabouik** [d99kris/nchat#156] [Ideas]
A a "Jump to quoted message" keybind when selecting a message containing a quote would be great. I guess the function could be partially re-used in the future to jump to matching results when a search feature is implemented.

**Comment by d99kris** [comment]
Hi @Kabouik,

Thanks for suggesting a new feature. I think it's a good idea. I'm currently a bit busy, but I will try to look into it
soon.

In any case, I encourage anyone else interested in this feature to upvote it. This will help gauge the level of
interest.

**Comment by d99kris** [comment]
Hi - support for jumping to quoted message has been implemented in 60983eb. Select a message (which is quoting another message) and press `alt-q` to jump to the quoted message.

**Comment by Kabouik** [comment]
Awesome!
